fe2-android/docs/README.md

```diff
@@ -0,0 +1,57 @@
+### PoP Android Frontend
+
+This repository contains the UI side implementation of the PoP project.
+
+#### Table of contents
+
+* [Architecture](#architecture) 
+* [Coding Standards](#coding-standards)
+
+#### Architecture
+
+We assume that you're familiar with the PoP project and the [User Interface Specifications](https://docs.google.com/document/d/1aVsCYj1vrTxh-V6CBrU4tRGANrC2wve47MSIXPliVbU/edit).
+
+##### Design
+
+The application follows the Model-View-ViewModel (MVVM) pattern and uses this [guide to app architecture](https://developer.android.com/jetpack/guide). 
+
+- The **View** consists of all the activities and fragments of the application. In this project there are only two activities respectively representing the detail and home view. The fragments represent a part of the UI within one of those activities. The packages are separated by views, for example the fragment for launching a LAO is in the home package. 
+
+- The **ViewModel** implements the UI logic and prepares and manages the data used by the activities and fragments. We use [LiveData](https://developer.android.com/topic/libraries/architecture/livedata), an observable data holder, to keep the UI data updated. In this project each activity has its own view model.
+- The **Model** consists of the local and remote data source, the model classes and the repository. The repository eases the data retrieval for the rest of the application. It is a mediator between the view model and the different data sources. In this application there are two data sources, the database and the PoP backend. The [Room](https://developer.android.com/reference/androidx/room/package-summary) persistence library is used to define the application's database, the entities and Data Access Object. 
```

Comment:
`the repository` previously referred to the Git repository. It might be worth specifying further what we're referring to here, possibly pointing to specific files.

`the entities` here have been referred to as `model classes`, `objects`, etc. 
It's important to have consistent naming across the documentation.